clippy_lints/src/methods.rs

```diff
@@ -487,11 +513,13 @@ impl LintPass for Pass {
                     FILTER_NEXT,
                     FILTER_MAP,
                     ITER_NTH,
-                    ITER_SKIP_NEXT)
+                    ITER_SKIP_NEXT,
+                    GET_UNWRAP)
     }
 }
 
 impl LateLintPass for Pass {
+    #[allow(cyclomatic_complexity)]
```

Comment:
Nevermind, I found `#[cyclomatic_complexity = "x"]`. It's a little weird though, and gives an `unused_attribute` warning, even though it is used. Is that a bug?